omnistrate_tests/test_standalone.py

```
@@ -2,6 +2,11 @@
 import signal
 from random import randbytes
 from pathlib import Path  # if you haven't already done so
+from redis.exceptions import (
+   ConnectionError,
```

Ruff

`redis.exceptions.ConnectionError` imported but unused

Remove unused import

(F401)